sqrl-tools/sqrl-config/src/main/java/com/datasqrl/config/ConnectorConfigImpl.java

```diff
@@ -27,7 +27,8 @@ public class ConnectorConfigImpl implements TableConfig.ConnectorConfig {
       "upsert-kafka", TableType.VERSIONED_STATE,
       "jdbc", TableType.LOOKUP,
       "jdbc-sqrl", TableType.LOOKUP,
-      "datagen", TableType.STREAM
+      "datagen", TableType.STREAM,
+      "postgres-cdc", TableType.STREAM
```

Comment:
Neither datagen or postgres-cdc should be here, it should be stream by default

Reply:
Wouldn't it be more beneficial to be explicit in this case? If not, why do we specify filesystem and Kafka? I was simply trying to follow an existing pattern.
If this is the default, shouldn't we change the loglvl of this log?
```
15:09:41.229 [main] WARN  com.datasqrl.config.ConnectorConfigImpl - Unrecognized connector type: datagen. Defaulting to STREAM table for import.
15:09:41.238 [main] WARN  com.datasqrl.config.ConnectorConfigImpl - Unrecognized connector type: postgres-cdc. Defaulting to STREAM table for import.
```